packages/terra-search-field/src/SearchField.jsx

```diff
@@ -273,17 +272,17 @@ class SearchField extends React.Component {
     return (
       <div {...customProps} className={searchFieldClassNames}>
         <div className={cx('input-group')}>
-          <Input
+          <input
             className={cx('input')}
             type="search"
             placeholder={placeholder}
             onChange={this.handleTextChange}
             disabled={isDisabled}
-            ariaLabel={inputText}
+            aria-label={inputText}
             aria-disabled={isDisabled}
             onKeyDown={this.handleKeyDown}
             onInput={this.handleInput}
-            refCallback={this.setInputRef}
+            ref={this.setInputRef}
             {...additionalInputAttributes}
```

Comment:
It maybe not be part of this scope, but in general we spread pass-through attributes before our implementation specific ones so the behaviors cannot be overridden. So I'd like to see this moved.

Reply:
Thanks @dkasper-was-taken  - logged this new issue here to move the spread to first in the list: https://github.com/cerner/terra-core/issues/3086